src/components/PeerGroupsUpdate.tsx

```diff
@@ -0,0 +1,176 @@
+import React, {useEffect, useRef, useState} from 'react';
+import {useDispatch, useSelector} from "react-redux";
+import {RootState} from "typesafe-actions";
+import { actions as peerActions } from '../store/peer';
+import {
+    Col,
+    Row,
+    Typography,
+    Space,
+    Button, Drawer, Form, Select, Tag
+} from "antd";
+import type { CustomTagProps } from 'rc-select/lib/BaseSelect'
+import {useAuth0} from "@auth0/auth0-react";
+import {PeerGroupsToSave} from "../store/peer/types";
```

Comment:
Codacy has a fix for the issue: [Replace `PeerGroupsToSave` with `·PeerGroupsToSave·`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
import { PeerGroupsToSave } from "../store/peer/types";
```